src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -621,7 +637,15 @@ private static void ValidateArrayResults(JsonDocument actualJsonDoc, JsonDocumen
                 actualElement.TryGetProperty(fieldName, out JsonElement actualValue);
                 expectedElement.TryGetProperty(fieldName, out JsonElement expectedValue);
 
-                if (fieldName.StartsWith(DATETIME_TYPE.ToLower()))
+                if (fieldName.StartsWith(DATETIMEOFFSET_TYPE.ToLower()))
+                {
+                    DateTimeOffset actualDateTimeOffset = DateTimeOffset.Parse(actualValue.ToString(), DateTimeFormatInfo.InvariantInfo, DateTimeStyles.AssumeUniversal);
```

Comment:
The actual comparison of the string values in this ValidateArrayResults and the above Compare* functions seem to be same. Can we move into helper functions?